todo.md

```diff
@@ -0,0 +1,3 @@
++ I want to be able to supply email and password on the pretty landing page.  
```

Comment:
Please remove and open Github Issues on the Project. 

https://guides.github.com/features/issues/ (as you close them, you get Github credit. Excellent practice!)